src/Sentry/Internal/DebugStackTrace.cs

```diff
@@ -88,25 +91,55 @@ internal void MergeDebugImagesInto(SentryEvent @event)
         Dictionary<string, string> relocations = new();
```

Comment:
We only get this far if there are already debug images on the event... from the comments above, it seems that only ever happens when there are InnerExceptions (and so multiple stack traces) - correct?

I guess a high level question at that point: What is the information that could overlap from one stack trace to another, in this scenario and how/what are we trying to merge here?